lib/epicenter_web/controllers/import_controller.ex

```diff
@@ -3,7 +3,7 @@ defmodule EpicenterWeb.ImportController do
   LiveView doesn't currently support file uploads, so this is a regular controller. It could probably be enhanced
   with some ajax or something.
   """
-
+  use Phoenix.VerifiedRoutes, endpoint: EpicenterWeb.Endpoint, router: EpicenterWeb.Router
```

Comment:
Do we need this here if we have it in `EpiCenterWeb.controller/0`?

Reply:
Same for the rest, I would think that handles all of the controllers so that we don't need the explicit `use` line in every controller.

Reply:
good point! removed